roles/install_wg/tasks/main.yml

```diff
@@ -31,6 +31,11 @@
         state: present
         key: https://www.elrepo.org/RPM-GPG-KEY-elrepo.org
 
+    - name: install fedora/epel GPG key for RHEL8
+      rpm_key:
+        state: present
+        key: https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
```

Comment:
Can we just add this key to the previous command that uses the `rpm_key` module? Or just do a loop with items with the previous command?

Reply:
e2d1c50

Reply:
Thanks, please squash

Reply:
you need to approve and merge, since CI is not yet configured here